declarative-pipeline-migration-assistant/src/main/webapp/js/prism.js

```diff
@@ -1101,7 +1101,7 @@ Prism.hooks.add('wrap', function(env) {
 		var hash = location.hash.slice(1);
 
 		// Remove pre-existing temporary lines
-		$$('.temporary.line-highlight').forEach(function (line) {
```

Comment:
Why are you changing this? According to https://github.com/jenkinsci/declarative-pipeline-migration-assistant-plugin/blob/master/declarative-pipeline-migration-assistant/src/main/webapp/js/prism.js#L984 this is an internally defined function and has nothing to do with prototype... Have you tested the plugin without prototype and found something broken?

Reply:
Yes, I have tested this plugin without prototype. Nothing was breaking. Thank you for the clarification.